### DISCUSSION TITLE
zh/guide/advanced/user

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#77] [Announcements]
# zh/guide/advanced/user

用户名 登录用户名 密码 登录密码 基本路径 用户登录时看到的根路径 允许 可以看到隐藏：可以看到隐藏的文件和文件夹; 无密码访问：无需密码即可访问; 添加 aria2 任务：可以添加 aria2 任务; 创建目录或上传：可以创建目录或上传文件; 重命名：可以重命名文件和文件夹; 移动：可以移动文件和文件夹; 复制：可以复制文件和文件夹; 删除：可以删...

https://alist.nn.ci/zh/guide/advanced/user.html

**Comment by baicaiaichibaicai** [comment]
guest能否支持访问多个目录？

**Comment by HAN915** [comment]
打开网页，不需要登录就能看到所有目录，这样正常吗？

**Reply by lklk1904** [reply]
在元信息里设置密码

**Reply by SkysCrystal** [reply]
没设置隐藏就能看到，但打不开（设置密码的文件夹）

**Comment by bian2022** [comment]
我想问一下github中的授权回调地址怎么填？

**Comment by qq727974** [comment]
啥时候支持微云呀

**Comment by iog666** [comment]
啥时候支持飞鸟呀

**Comment by Alibaba404** [comment]
能否支持一个用户允许添加多个路径,这些路径可能存在于不同的存储/磁盘上,该用户登录后就能看到设置的目录

**Comment by qifeibro** [comment]
tips第3条，我复制文件很多，不知道是不是这个原因导致最后alist断开连接，并且不能重新启动，最后只得卸载重新安装了。

**Comment by 957176113** [comment]
百度网盘无法使用 提示：{"error_code":31329,"error_msg":"hit black userlist , hit illeage dlna","request_id":247293419898638771}

**Comment by battlewolf9527** [comment]
用户设置里的无需密码访问究竟有什么用？我设置了一个用户无需密码访问，同时设置了密码，登录时依然需要密码；然后我又添加了一个用户没有设置密码，但不勾选无需密码访问，登录时就不需要密码，似乎这个设置不工作？

**Comment by nishizhu123456** [comment]
1

**Comment by Pterosaur** [comment]
如果想用户只有通过WebDAV上传创建文件的权限而没有删除的权限，应该如何设置？我必须启用WebDAV 管理，不然用户没法创建文件夹，但是禁用了删除，用户还是可以删除文件。

**Comment by 7788user** [comment]
hello
